OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -320,10 +319,10 @@ protected function configureOptions(OptionsResolverInterface $resolver)
         ));
 
         $resolver->setDefaults(array(
-            'scope'               => null,
```

Comment:
Personally I really don't like this change as it makes arrays much less readable...